lib/chef/provisioning/docker_driver/docker_transport.rb

```diff
@@ -236,40 +150,10 @@ def using_boot2docker?
       end
     end
 
-    # Copy of container.attach with timeout support and pipeline
-    def attach_with_timeout(container, read_timeout, options = {}, &block)
-      opts = {
-        :stream => true, :stdout => true, :stderr => true
-      }.merge(options)
-      # Creates list to store stdout and stderr messages
-      msgs = Docker::Messages.new
-      connection.start_request(
-        :post,
-        "/containers/#{container.id}/attach",
-        opts,
-        :response_block => attach_for(block, msgs),
-        :read_timeout => read_timeout,
-        :pipeline => true,
-        :persistent => true
-      )
-    end
-
-    # Method that takes chunks and calls the attached block for each mux'd message
-    def attach_for(block, msg_stack)
-      messages = Docker::Messages.new
-      lambda do |c,r,t|
-        messages = messages.decipher_messages(c)
-        msg_stack.append(messages)
-
-        unless block.nil?
-          messages.stdout_messages.each do |msg|
-            block.call(:stdout, msg)
-          end
-          messages.stderr_messages.each do |msg|
-            block.call(:stderr, msg)
-          end
-        end
-      end
+    def container_path(path)
+      docker_root = File.join(Docker.info['DockerRootDir'], %w(aufs mnt))
```

Comment:
ok. good to know. There must be a way to get this dynamically. I'll figure that out and fix.